app/src/main/java/com/github/dodobest/upbitapi/UpbitViewModel.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.dodobest.upbitapi
+
+import android.text.Editable
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.github.dodobest.domain.model.UpbitMarketData
+import com.github.dodobest.domain.model.UpbitTickerData
+import com.github.dodobest.domain.usecase.GetMarketsUseCase
+import com.github.dodobest.domain.usecase.GetTickerUseCase
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+
+class UpbitViewModel(
+    private val getMarketsUseCase: GetMarketsUseCase,
+    private val getTickerUseCase: GetTickerUseCase,
+) : ViewModel() {
+    private val _marketData = MutableLiveData<List<UpbitMarketData>>()
+    val marketData: LiveData<List<UpbitMarketData>>
+        get() = _marketData
+
+    private val _tickerData = MutableLiveData<Map<String, UpbitTickerData>>()
+    val tickerData: LiveData<Map<String, UpbitTickerData>>
+        get() = _tickerData
+
+    private val _tickerSearchName = MutableLiveData<String>()
+    val tickerSearchName: LiveData<String>
+        get() = _tickerSearchName
+
+    private val _errMessage = MutableLiveData<String>()
+    val errMessage: LiveData<String>
+        get() = _errMessage
+
+    fun setTickerSearchName(inputSearchName: Editable) {
+        _tickerSearchName.value = inputSearchName.toString()
+    }
+
+    fun getMarkets() {
+        getMarketsUseCase.execute().observeOn(AndroidSchedulers.mainThread())
```

Comment:
```suggestion
        getMarketsUseCase.execute()
            .observeOn(AndroidSchedulers.mainThread())
```
함수 체이닝 컨벤션을 지켜주세요. (이하 관련 내용 동일)

Reply:
[19a41df](https://github.com/DoTheBestMayB/UpbitAPI/pull/21/commits/19a41dfc626e7b8f64c34f88ab52a21c8032dae9)